cluster/bin/snappy-job.sh

```diff
@@ -127,7 +127,7 @@ while (( "$#" )); do
         echo "The netrc file $passwordfile not found."
         exit 1
       fi
-      securePart=" --netrc-file ${passwordfile}"
+      securePart=" --config ${passwordfile}"
```

Comment:
Will config take only password file?
If so, a more intuitive name will be better. But if you are not able able to work can be take care in documentation.
If not (means it will take other configuration parameters using --config option) then I am confused how that would work.